src/test/java/com/exasol/glue/ittests/BaseIntegrationTest.java

```diff
@@ -62,7 +91,19 @@ public String getPassword() {
     }
 
     public Map<String, String> getDefaultOptions() {
-        return Map.of("jdbc_url", getJdbcUrl(), "username", getUsername(), "password", getPassword());
+        final Map<String, String> map = new HashMap<>(Map.of( //
+                "jdbc_url", getJdbcUrl(), //
+                "username", getUsername(), //
+                "password", getPassword(), //
+                "awsAccessKeyId", S3.getAccessKey(), //
+                "awsSecretAccessKey", S3.getSecretKey(), //
+                "awsRegion", S3.getRegion(), "s3Bucket", DEFAULT_BUCKET_NAME, //
+                "s3PathStyleAccess", "true", //
+                "awsEndpointOverride", "localhost:" + S3.getMappedPort(4566)));
```

Comment:
This won't work when the docker is running in a VM (e.g. on macOS). Use docker host IP instead of localhost.

Reply:
Looks good!